modules/configuring-cluster-loader.adoc

```diff
@@ -0,0 +1,233 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/using-cluster-loader.adoc
+
+[id='configuring_cluster_loader_{context}']
+= Configuring Cluster Loader
+
+Create multiple namespaces (projects), which contain multiple templates or pods.
+
+Locate the configuration files for Cluster Loader in the `config/` subdirectory.
+The pod files and template files referenced in these configuration examples are
+found in the `content/` subdirectory.
+
+== Configuration fields
+
+.Top-level Cluster Loader Fields
+|===
+|Field |Description
+
+|`cleanup`
+|Set to `true` or `false`. One definition per configuration. If set to `true`,
+`cleanup` will delete all namespaces (projects) created by Cluster Loader at the
+end of the test.
+
+|`projects`
+|A sub-object with one or many definition(s). Under `projects`, each
+namespace to create is defined and `projects` has several mandatory subheadings.
+
```

Comment:
I have no idea what this means!

Reply:
@sjug Can you help us clarify this? There are a few other questions below as well. Thanks!